assets/queries/cloudFormation/ec2_network_ACL_deny_rules/test/positive_expected_result.json

```diff
@@ -0,0 +1,7 @@
+[
+	{
+		"queryName": "EC2 network ACL denied traffic",
+		"severity": "MEDIUM",
+		"line": 17
+	}
+]
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline